kuma/search/filters.py

```diff
@@ -16,6 +20,17 @@ def get_filters(getter_func):
     return filters.keys()
 
 
+class QueryStringValidationMixin(object):
+    """
+    A mixin to validate query string parameters.
+    """
+    def validate_query_params(self, request):
```

Comment:
Ooh yeah, good call. That will also remove the need for this mixin. I thought that was a bad code smell.